share/setup.nsi.in

```diff
@@ -20,7 +20,7 @@ SetCompressor /SOLID lzma
 !define MUI_STARTMENUPAGE_REGISTRY_KEY ${REGKEY}
 !define MUI_STARTMENUPAGE_REGISTRY_VALUENAME StartMenuGroup
 !define MUI_STARTMENUPAGE_DEFAULTFOLDER "@PACKAGE_NAME@"
-!define MUI_FINISHPAGE_RUN $INSTDIR\bitcoin-qt.exe
+!define MUI_FINISHPAGE_RUN $INSTDIR\@PACKAGE_TARNAME@-qt.exe
```

Comment:
I agree that that would be useful.  Unfortunately my familiarity with autoconf is limited enough that I'm not fully comfortable implementing that myself.  Maybe someone else could do that as another pull request?